src/main/xerus/monstercat/api/Player.kt

```diff
@@ -114,7 +114,11 @@ object Player: FadingHBox(true, targetHeight = 25) {
 	fun reset() {
 		fadeOut()
 		if(!Files.isDirectory(Settings.PLAYEREXPORTFILE())) {
```

Comment:
What if it is a directory? Doesn't look like that's handled

Reply:
Why should it need to? If it's a directory, we *could* show an error message but that would be horrible for the user as it'll show each time a write is made

Reply:
But silent failure seems like unexpected behavior as well. How about at least adding a debug log message?

Reply:
How about adding an extension to the file in case it's already a folder?